CIP-0072/README.md

```diff
@@ -0,0 +1,382 @@
+              "type": "array",
+              "items": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "version": {
+                      "type": "integer"
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "version"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "releaseNumber",
+            "releaseName",
+            "auditId",
+            "scripts"
+          ]
+        }
+      ]
+    },
+    "scripts": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "name": {
+              "type": "string"
+            },
+            "purpose": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string"
+            },
+            "versions": {
+              "type": "array",
+              "items": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "version": {
+                      "type": "integer"
+                    },
+                    "plutusVersion": {
+                      "type": "integer"
+                    },
+                    "fullScriptHash": {
+                      "type": "string"
+                    },
+                    "scriptHash": {
+                      "type": "string"
+                    },
+                    "contractAddress": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "version",
+                    "plutusVersion",
+                    "fullScriptHash",
+                    "scriptHash",
+                    "contractAddress"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "id",
+            "name",
+            "purpose",
+            "type",
+            "versions"
+          ]
+        }
+      ]
+    }
+  },
+  "required": [
+    "subject",
+    "projectName",
+    "link",
+    "social",
+    "category",
+    "subCategory",
+    "description",
+    "releases",
+    "scripts"
+  ]
+}
+```
+
+### Example
+
+```json
+{
+  "subject": "9SYAJPNN",
+  "projectName": "My Project",
+  "link": "https://myProject.app",
+  "social": {
+    "twitter": "twiterHandle",
+    "github": "githubHandle",
+    "website": "https://website"
+  },
+  "category": "GAMING",
+  "subCategory": "RPG",
+  "description": {
+    "short": "A story rich game where choices matter"
+  },
+  "releases": [
+    {
+      "releaseNumber": 1,
+      "releaseName": "V1",
+      "auditId": "z5L90f",
```

Comment:
I thought we agreed to remove audits?

Reply:
Ah yes; missed this one. Thanks @matiwinnetou